src/main/resources/application-metering-job.yaml

```diff
@@ -12,5 +12,6 @@ rhsm-subscriptions:
         jobBackOffMaxInterval: ${METERING_JOB_BACK_OFF_MAX_INTERVAL:50000}
         jobBackOffInitialInterval: ${METERING_JOB_BACK_OFF_INITIAL_INTERVAL:1000}
         jobBackOffMultiplier: ${METERING_JOB_BACK_OFF_MULTIPLIER:1.5}
+        eventSource: ${EVENT_SOURCE}
```

Comment:
Since the `swatch-metrics-sync` (uses this metering-job profile) doesn't produce messages to the `platform.rhsm-subscriptions.service-instance-ingress` topic, I don't think it needs to have an event_source yet since it won't be used.  Can also remove it from the job definition in the clowdapp.yaml file (lines 488)

Reply:
Removed